recipes/qhull/all/patches/no-cpp11.patch

```diff
@@ -1,11 +1,19 @@
 --- a/src/libqhullcpp/QhullUser.cpp
 +++ b/src/libqhullcpp/QhullUser.cpp
-@@ -98,7 +98,7 @@ captureOff()
+@@ -11,6 +11,7 @@
+ #include "libqhullcpp/QhullError.h"
+ 
+ #include <iostream>
++#include <stdint.h>
+ 
+ using std::cerr;
+ using std::endl;
+@@ -98,7 +99,7 @@ captureOff()
          throw QhullError(10080, "Qhull error: QhullUser::captureOn not call before QhullUser::captureOff for QhullUser 0x%llx", 0, 0, 0.0, this);
      }
      if(qh()->cpp_user!=this){
 -        throw QhullError(10081, "Qhull error: conflicting QhullUser (0x%llx) for QhullUser::captureOff().  Does not match 'this' (0x...%X)", int((uintptr_t)this), 0, 0.0, qh()->cpp_user);
-+        throw QhullError(10081, "Qhull error: conflicting QhullUser (0x%llx) for QhullUser::captureOff().  Does not match 'this' (0x...%X)", *(int *)this, 0, 0.0, qh()->cpp_user);
```

Comment:
it's not safe to use int instead of intptr_t, as pointer size could be greater than int. intptr_t is the right type to store the pointer.

Reply:
Correct, I'll fix this patch.

Reply:
Actually it's already fixed:

```patch
+        throw QhullError(10081, "Qhull error: conflicting QhullUser (0x%llx) for QhullUser::captureOff().  Does not match 'this' (0x...%X)", int(0xffff&(intptr_t)this), 0, 0.0, qh()->cpp_user);
     }
```